cdap-security-spi/src/main/java/co/cask/cdap/security/spi/authorization/AbstractAuthorizer.java

```diff
@@ -62,16 +65,18 @@ public void enforce(EntityId entity, Principal principal, Action action) throws
   }
 
   @Override
-  public Predicate<EntityId> createFilter(Principal principal) throws Exception {
-    Set<Privilege> privileges = listPrivileges(principal);
-    final Set<EntityId> allowedEntities = new HashSet<>();
-    for (Privilege privilege : privileges) {
-      allowedEntities.add(privilege.getEntity());
-    }
+  public Predicate<EntityId> createFilter(final Principal principal) throws Exception {
```

Comment:
Seems like this method should not have `throws Exception`, since it just create a filter.